TestKoinGraph > checking koin modules FAILED     java.lang.NoClassDefFoundError at Shadows.java

When compiling the app from Android Studio (AF 2020.3.1 RC1), I am facing this error.
```
> Task :app:testDebugUnitTest FAILED

com.surrus.peopleinspace.TestKoinGraph > checking koin modules FAILED
    java.lang.NoClassDefFoundError at Shadows.java:2317
        Caused by: java.lang.ClassNotFoundException at SandboxClassLoader.java:147
            Caused by: java.lang.IllegalArgumentException at ClassReader.java:195
```

Not sure what is causing this.

--- a/buildSrc/src/main/java/Dependencies.kt
+++ b/buildSrc/src/main/java/Dependencies.kt
@@ -18,10 +18,10 @@ object Versions {
     const val accompanist = "0.14.0"
 
     const val junit = "4.12"
-    const val mockito = "3.7.7"
+    const val mockito = "3.11.2"
     const val testCore = "1.3.0"
     const val testRunner = "1.3.0"
-    const val robolectric = "4.4"
+    const val robolectric = "4.6.1"
 
     const val sqlDelight = "1.5.0"
     const val shadow = "7.0.0"

